js/apps/admin-ui/src/clients/ClientsSection.tsx

```diff
@@ -71,6 +73,15 @@ const ClientDetailLink = (client: ClientRepresentation) => {
           </Badge>
         )}
       </Link>
+      {client.attributes?.["temporary_admin"] == "true" && (
+        <Tooltip content={t("temporaryService")}>
+          <WarningTriangleIcon
+            style={{ color: "gold" }}
```

Comment:
I don't think the style is needed.  Default should be fine?

Reply:
@ssilvert I don't have a strong opinion here, I just tried to match the [gold colour](https://github.com/keycloak/keycloak/pull/31387/commits/8b2572e14d57a521f59b8992549367d61a7b4d37#diff-3c2dbdf23139622b8c5770cbaa5a44d98cf4437a9745a71ffafbf01eb37bc8ceR10) that @vmuzikar decided to use  earlier for the banner.

Reply:
@Pepo48 @vmuzikar I suggest going with the default for both.  That way, the "warning" color is consistent throughout the application.

Reply:
Addressed.

Reply:
@Pepo48 I can see the banner now grey. I believe it's because we're not using the suggested `pf-v5-global--warning-color--100` color maybe?

Reply:
Oh, I thought that was addressed.  

@Pepo48 @vmuzikar Yes, we should set the banner color properly.